app/api/auth/create_access_token.go

```diff
@@ -32,31 +32,59 @@ const parsedRequestData ctxKey = iota
+//				because if token are only stored in cookies, the frontend never has the hand on it and so does not know
+//				on launch whether the user is logged or not.
 //
-//		If the `{code}` is not given while the "Authorization" header or/and the "access_token" is given
-//		(when both are given, the "Authorization" header is used and the cookie gets deleted),
-//		the service refreshes the access token
-//		(locally for temporary users or via the login module for normal users) and
-//		saves it into the DB keeping only the input token and the new token.
-//		Since the login module responds with both access and refresh tokens, the service updates the user's
-//		refresh token in this case as well. If there is no refresh token for the user in the DB,
-//		the 'not found' error is returned.
+//		The `{code}` parameter is an output of the login-module after a successful login.
+//
+//		* If the `{code}` is given and the "Authorization" header is not given.
+//			This happens after the client successfully logged on the login-module.
+//			Then,
+//			the service converts the given OAuth2 authorization code into tokens,
+//			creates or updates the authenticated user in the DB with the data returned by the login module,
+//			and saves new access & refresh tokens into the DB as well.
+//			If OAuth2 authentication has used the PKCE extension, the `{code_verifier}` should be provided
+//			so it can be sent together with the `{code}` to the authentication server.
+//
+//
+//		* If the `{code}` is not given while the "Authorization" header or/and the "access_token" cookie is given
+//			(when both are given, the "Authorization" header is used, and the cookie gets deleted),
+//			and if the authentication is valid.
+//			This happens when the frontend app loads and the user is already logged-in.
+//			Then,
+//			the service refreshes the access token
+//			(locally for temporary users or via the login module for normal users) and
+//			saves it into the DB keeping only the input token and the new token.
+//			Since the login module responds with both access and refresh tokens, the service updates the user's
+//			refresh token in this case as well.
+//			If there is no refresh token for the user in the DB,
+//			the 'not found' error is returned.
+//
+//
+//		* If the `{code}` is not given,
+//			and if the authentication is not given (no "Authorization" header and no "access_token" cookie) or is invalid.
+//			This happens when the frontend app loads, and the user is not logged-in, or if the authentication
+//			is not valid anymore.
+//			Then,
+//			we create a temporary user and log-in the user as it.
 //
 //
 //		If attributes of the old and the new 'access_token' cookies are different (or the token is returned in the JSON),
 //		the old cookie gets deleted (otherwise, just overwritten).
 //
+//		If a cookie is given together with a `{code}`, the cookie is deleted.
+//
+//		`{default_language}` is used only if a temporary user is created.
```

Comment:
If `{default_language}` is missing ? I updated to add that is uses `users.default_language` as default. 

Or is if the cookie is missing? This case is defined above, but maybe it's clear enough.

Reply:
For the new case of this PR: when it creates a temp user because there is no valid auth...  you need to know what is the default language of the new user

Reply:
ok, then it is fixed 👍